python/django/security/injection/reflected-data-httpresponse.yaml

```diff
@@ -68,107 +68,7 @@ rules:
             ...
             django.http.HttpResponse(..., $INTERM, ...)
         - pattern: $A = django.http.HttpResponse(..., request.$W.get(...), ...)
```

Comment:
Looks like some of the one-liners got missed; either way, this still looks good!

Reply:
yeah the linter doesn't find those yet.